hack/update-codegen.sh

```diff
@@ -24,8 +24,8 @@ function make_aliases() {
   # self-contained and we don't release them.
   ( cd mechanics/tools/gen-aliases; go build . )
   # Auto-generate OWNERS_ALIASES from peribolos configs
-  ./mechanics/tools/gen-aliases/gen-aliases knative peribolos/knative.yaml OWNERS_ALIASES
-  ./mechanics/tools/gen-aliases/gen-aliases knative-sandbox peribolos/knative-sandbox.yaml knative-sandbox-OWNERS_ALIASES
+  ./mechanics/tools/gen-aliases/gen-aliases knative peribolos/knative.yaml peribolos/knative-OWNERS_ALIASES
+  ./mechanics/tools/gen-aliases/gen-aliases knative-sandbox peribolos/knative-sandbox.yaml peribolos/knative-sandbox-OWNERS_ALIASES
```

Comment:
do we need an entry somewhere for:
peribolos/knative-sandbox-OWNERS_ALIASES

Reply:
Wouldn't let me comment on the entry in OWNERS_ALIASES file so just added it here.

Reply:
I think this has entries for generating both files.

Reply:
I didn't include it because I was thinking of just updating the sync actions here:

https://github.com/knative-sandbox/knobots/blob/054b91b98bca48e7b42227e31f0590269b3a794d/actions/update-community/entrypoint.sh#L31-L35